code/modules/mob/living/carbon/human/say.dm

```diff
@@ -176,7 +182,7 @@
 			var/obj/item/device/radio/R = get_radio()
 			if(R)
 				used_radios += R
-				if(R.talk_into(src, message, null, verb, speaking))
+				if(R.talk_into(src, message, isAdminLevel(z) ? "department" : null, verb, speaking))
```

Comment:
You should update the examine text on antag radios at least do people know about this feature

Reply:
Added. When a syndie headset (Burg, merc, ninja, etc) is initialized, it'll give `desc_antag` to let them know.